src/Misc/index.ts

```diff
@@ -55,3 +55,4 @@ export * from './timer';
 export * from "./copyTools";
 export * from "./reflector";
 export * from "./domManagement";
+export * from "./PerformanceViewer/index";
```

Comment:
no need for index

Reply:
Oh huh, weird it wasn't working without index before but it is now (I had thought it wouldn't need the index either). Thanks for the suggestion!

Reply:
Updated!

Reply:
[22:18:48] Line 58 Exports ./PerformanceViewer/ should be from the full path including index. /home/vsts/work/1/s/src/Misc/index.ts 

If I remove the index it seems to fail the lint check :(

Reply:
Sorry! my fault. index is needed...

Reply:
Index is back!